scripts/fix-ca-for-k3s.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage:
+# % ./scripts/fix-ca-for-k3s.sh [member-cluster] ...
+#
+# Description:
+# This script fixes up the configuration for k3s.
+# Namely it updates ca.crt in secrets to match with the ones in
+# KUBECONFIG. It's intended to be run after deploy-federation.sh
+# script.
+#
+# Background:
+# In k3s, different endpoints and certificates are configured for
```

Comment:
what is k3s here? its confusing.

Reply:
this k3s it is. https://github.com/rancher/k3s

Reply:
I think @shashidharatd's comments suggest including a link to the project.

Reply:
yes, and i did.

Reply:
Ah, sorry.  I was expecting a change to this line but I see you updated the paragraph above.  I do so love github review.